core/src/miner/backup.rs

```diff
@@ -0,0 +1,96 @@
+// Copyright 2019 Kodebox, Inc.
+// This file is part of CodeChain.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+use std::collections::HashMap;
+
+use ckey::Public;
+use kvdb::{DBTransaction, KeyValueDB};
+use primitives::H256;
+use rlp::Encodable;
+
+use super::mem_pool_types::{MemPoolItem, PoolingInstant};
+use crate::db as dblib;
+
+const PREFIX_SIZE: usize = 5;
+const ITEM: &[u8; PREFIX_SIZE] = b"item_";
+const SEQS: &[u8; PREFIX_SIZE] = b"seqs_";
```

Comment:
I think `PREFIX_ITEM` and `PREFIX_SEQS` may be better. Too short names make readers confused.

Reply:
Fixed